einterfaces/message_export.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package einterfaces
+
+import (
+	"github.com/mattermost/mattermost-server/model"
+)
+
+type MessageExportInterface interface {
+	StartSynchronizeJob(waitForJobToFinish bool) (*model.Job, *model.AppError)
```

Comment:
Not sure about the naming of this function. Perhaps `StartExportJob()` would be better?

Reply:
I was following the example in the LDAP code. It has "Synchronize" in the name because it blocks while waiting for the job to finish.